lib/Word_Lib/Word_32.thy

```diff
@@ -172,6 +172,10 @@ lemma word32_and_max_simp:
   using word_and_full_mask_simp [of x]
   by (simp add: numeral_eq_Suc mask_Suc_exp)
 
+lemma max_minus_one_word32:
```

Comment:
We should name both of them just `max_minus_one_word`, so that we can use the same name in 32 and 64 contexts (maybe with another postfix for indicating that it is for fixed lengths, not sure).

The theory name can be used to specify a specific version if that is needed.